CodeWars/Tasks/BubbleSortAlgorithm.cs

```diff
@@ -4,9 +4,9 @@ namespace CodeWars.Tasks
     /// <summary>
     /// https://www.codewars.com/kata/56b97b776ffcea598a0006f2/train/csharp
```

Comment:
Nice try)))
Lets create separete file for BubbleSortOnceResolve and BubbleSortAlgorithm. 
Please, remove codewars link as it is not refer to BubbleSortAlgorithm.